docs/pages/global.css

```diff
@@ -257,3 +257,10 @@ div.nextra-search kbd {
     opacity: 0;
   }
 }
+
+ul ul, 
+ul ol,
+ol ul,
+ol ol {
+  @apply mb-6;
+}
```

Comment:
Please let me know if there's a better way to apply this bottom margin to the nested lists.